doc/source/_toc.yml

```diff
@@ -213,9 +213,9 @@ parts:
               - file: serve/advanced-guides/inplace-updates
               - file: serve/advanced-guides/dev-workflow
               - file: serve/advanced-guides/deployment-graphs
-              - file: serve/advanced-guides/direct-ingress
               - file: serve/advanced-guides/managing-java-deployments
               - file: serve/advanced-guides/deploy-vm
+              - file: serve/advanced-guides/grpc-guide
```

Comment:
I think in ordering this should be above deployment graphs (the other features are strictly more experimental/less important)

Reply:
oh cool, did know there is an implication for the ordering. Will move this up!